benchmarks/src/main/java/org/corfudb/benchmarks/runtime/collections/experiment/rocksdb/RocksDbMap.java

```diff
@@ -0,0 +1,259 @@
+package org.corfudb.benchmarks.runtime.collections.experiment.rocksdb;
+
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import lombok.Builder;
+import lombok.Builder.Default;
+import lombok.Getter;
+import lombok.NonNull;
+import org.corfudb.protocols.wireprotocol.ICorfuPayload;
+import org.rocksdb.Options;
+import org.rocksdb.PlainTableConfig;
+import org.rocksdb.RocksDB;
+import org.rocksdb.RocksDBException;
+import org.rocksdb.RocksIterator;
+import org.rocksdb.WriteOptions;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * A persistent Map backed by RocksDb embedded database.
+ *
+ * @param <K> key type
+ * @param <V> value type
+ */
+@Builder
+public class RocksDbMap<K, V> implements Map<K, V> {
+    private static final String TMP_DIR = System.getProperty("java.io.tmpdir");
+
+    static {
+        RocksDB.loadLibrary();
+    }
+
+    @Getter
+    private RocksDB db;
+
+    @NonNull
+    private final Class<K> keyType;
+
+    @NonNull
+    private final Class<V> valueType;
+
+    @NonNull
+    @Default
+    @Getter
+    private final Path dbPath = Paths.get(TMP_DIR, "corfu", "rt", "persistence", "rocks_db");
+
+    @NonNull
+    @Default
+    private final WriteOptions opts = new WriteOptions().setDisableWAL(true);
+
+    @NonNull
+    @Default
+    private final Options options = new Options()
+            .setCreateIfMissing(true)
+            .setWriteBufferSize((long) 256 * (long) 1024 * (long) 1024)
+            .setMaxWriteBufferNumber(4)
+            .setIncreaseParallelism(8)
+            //.setBloomLocality(10)
+
+            //.setOptimizeFiltersForHits(true)
```

Comment:
Remove commented code.

Reply:
Not sure if we should remove it. 
1. Those parameters can be used to adjust rocksDb performance. They can be used for a quick check of how the DB behaves with different settings.
2. Since it's not production code I believe it's ok to have those settings commented.
3. There is no rich documentation for rocksdb_jni library and it's not immediately obvious how to set up those parameters when you read documentation and see that settings for c++ version. The commented code can help/clarify that.